cms/envs/common.py

```diff
@@ -841,6 +841,8 @@
 
 MIDDLEWARE = [
     'openedx.core.lib.x_forwarded_for.middleware.XForwardedForMiddleware',
+    'edx_django_utils.security.csp.middleware.content_security_policy_middleware',
```

Comment:
Why snake case?

Reply:
Because it's a function middleware rather than a class middleware -- it ended up being simpler to write that way.

Reply:
Oops. Sorry I had missed your response. Makes sense. Didn't realize this was an option.